multicore_runtime/wsgi_config.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Utilities to configure the dispatcher, middleware and environment."""
+
+import logging
+import os
+import threading
+import UserDict
+
+from static_files import static_app_for_regex_and_files
+
+from google.appengine.api import appinfo
+from google.appengine.api import appinfo_includes
+from google.appengine.runtime import wsgi
+
+
+def get_module_config_filename():
+  """Returns the name of the module configuration file (app.yaml)."""
+  module_yaml_path = os.environ['MODULE_YAML_PATH']
```

Comment:
Are you OK with the `KeyError` if the environment variable is not present?

Perhaps add a `Raises:` section to your docstring?